platform/package.yaml

```diff
@@ -0,0 +1,70 @@
+name:                platform
+version:             0.1.0.0
+github:              "githubuser/platform"
+license:             BSD-3-Clause
```

Comment:
I'd be inclined to make this the MIT license if that's an option.

Reply:
done